recipes/coolbox/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "CoolBox" %}
-{% set version = "0.3.0" %}
+{% set version = "0.3.0.a" %}
```

Comment:
Is this an alpha release?

Reply:
Yes, it's alpha, but the '.a' is just for pass the lint, it need a different version number.